Modules/@babylonjs/react-native/android/src/main/java/com/reactlibrary/EngineViewManager.java

```diff
@@ -28,4 +31,30 @@ public void onDropViewInstance(@NonNull EngineView view) {
         super.onDropViewInstance(view);
         // TODO: Native view specific cleanup
     }
+
+    @Override
+    public Map<String,Integer> getCommandsMap() {
+        return MapBuilder.of(
+            COMMAND_TAKE_SNAPSHOT_NAME,
+            COMMAND_TAKE_SNAPSHOT
+        );
+    }
+
+    @Override
+    public void receiveCommand(final EngineView view, String commandId, ReadableArray args) {
+        // This will be called whenever a command is sent from react-native.
+        switch (commandId) {
+            case COMMAND_TAKE_SNAPSHOT_NAME:
+                view.takeSnapshot();
+                break;
```

Comment:
Should we throw in the `default` case?

Reply:
Examples I've seen do not do this, but I will throw this in there for safety